### PR TITLE
Docs update 3

### PR DESCRIPTION
**Pull request by zehawki**
The Inbuilt Plugins section of the docs is also updated.

### CODE REVIEW

**Comment by zehawki** [comment]
@macrozone I see that the diff for sidebar.md (https://github.com/react-page/react-page/pull/955/files#diff-2a55b1a76ffb4ba683697763144889d5759e3f6204a8b54e94d41435523c6f97) shows a `[Recipes & Tips](/recipes.md)` link, but I've never seen that in the docs...

**Comment by zehawki** [comment]
@macrozone this is still pending, BTW

**Comment by macrozone** [comment]
there are unanswered questions

**Comment by zehawki** [comment]
>there are unanswered questions

Sorry, didnt get. Where do I see these?

**Review by macrozone — CHANGES_REQUESTED** [review]

**Comment by macrozone** [comment]
ah sorry, forgot to click

File: docs/quick-start.md [hunk 1]
```diff
@@ -32,33 +31,11 @@ $ yarn add @material-ui/core @material-ui/styles @material-ui/icons
...
 
 React-page editor together with various plugins can be used to create and subsequently edit pages in a “WYSIWYG” mode. The editing UI code is lazy loaded in this case.
 
 ### Viewing
 
 The React-page editor can seamlessly be used to view the pages created by it in a "readonly" mode. The bundle size is reduced in this mode.
-
-[readonly.tsx](examples/pages/examples/readonly.tsx ':include :type=code typescript')
-
-(see demo here)](//demo/examples/readonly)
```
[the remark this answers is not in the document]
Comment (zehawki) [reply docs/quick-start.md:64]: 1. This section is "getting started" - it does not need to provide every bit of info
2. It does not make sense to have the same piece of code repeated in multiple places - over time, with different collaborators etc, these diverge and cause confusion in new users mind. This was the same problem I faced when I first started - the two pieces of code showed different things
3. A full, relevant example is just below, in the next section: Editor component

**Comment by zehawki** [comment]
@macrozone did you get a chance to see my replies? I'm afraid the longer the changes are not merged, the more they may conflict later.

**Review by macrozone — CHANGES_REQUESTED** [review]

**Comment by macrozone** [comment]
sorry i forgot to submit the review

**Pull request merged**